ietf-layer0-types-ext.yang

```diff
@@ -568,6 +568,31 @@ module ietf-layer0-types-ext {
     }
   }  
 
+  grouping pdl-penalty {
+    description "entries of table; pair of values
+    polarization dependent loss and
+    associated penalty";
+
+    leaf max-polarization-dependent-loss {
+      type decimal64 {
+        fraction-digits 2;
+      }
+      units dB ;

```

Comment:
1) Editorial comment on the unit statement
2) Should also this attribute be config false?
3) Should also this attribute be mandatory?

```suggestion
      units "dB";
      config false;
      mandatory true;
```